reana_server/utils.py

```diff
@@ -119,7 +118,7 @@ def publish_workflow_submission(workflow, user_id, parameters):
         validate_job_memory_limits(complexity)
     current_workflow_submission_publisher.publish_workflow_submission(
         user_id=str(user_id),
-        workflow_id_or_name=workflow.get_full_workflow_name(),
+        workflow_id_or_name=str(workflow.id_),
```

Comment:
Question: can you explain why this is needed?

Reply:
Because `scheduler` sends messages to `job-status-consumer` to fail workflows, and `job-status-consumer` needs UUID, I replaced it here with UUID.

Why `str`? Because, `workflow.id_` is not a string, I needed to convert it.

Reply:
I see, then probably it would be better to rename `workflow_id_or_name` to `workflow_uuid` (like in `WorkflowStatusPublisher`) to make it more clear what value we are sending

Reply:
Yep. Will update the other PR.